grails-app/domain/au/org/ala/ecodata/Project.groovy

```diff
@@ -175,6 +177,10 @@ class Project {
         externalIds.find{it.idType == ExternalId.IdType.INTERNAL_ORDER_NUMBER}?.externalId
     }
 
+    String getRaId() {
```

Comment:
Is this necessary?  It's adding a NESP specific derived property to the Project unnecessarily which will clutter the JSON response + API for every other use case.  (and it's not clear what the property actually is - the term RaId has no meaning in any context other than a NESP specific acryonm)
The exporter is not even using it (it's retrieving the value via the externalId which makes sense to me).